src/Modules/CalcDefence.lua

```diff
@@ -160,7 +160,7 @@ function calcs.defence(env, actor)
 		local gearArmour = 0
 		local gearEvasion = 0
 		local slotCfg = wipeTable(tempTable1)
-		for _, slot in pairs({"Helmet","Body Armour","Gloves","Boots","Weapon 2","Weapon 3"}) do
+		for _, slot in pairs({"Helmet","Gloves","Boots","Body Armour","Weapon 2","Weapon 3"}) do
```

Comment:
Had to change this order, otherwise mod doesn't exist since armour/es values for other pieces haven't been calculated yet.

Can potentially look at reworking to loop twice if worried this could cause any other issues. Otherwise this will most likely become tech debt in the case that a similar mod is added for any other piece of gear.